pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -118,45 +194,167 @@ func haveOverlap(a1, a2 []string) bool {
 }
 
 func (pl *VolumeRestrictions) PreFilter(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod) (*framework.PreFilterResult, *framework.Status) {
-	if pl.enableReadWriteOncePod {
-		return nil, pl.isReadWriteOncePodAccessModeConflict(ctx, pod)
+	s, err := pl.calPreFilterState(ctx, pod)
+	if err != nil {
```

Comment:
one of the failures could be for PVC NotFound, in which case we should return `UnschedulableAndUnresolvable`, instead of error.